### PR TITLE
Deploying a global policy for HoH local testing

### PR DESCRIPTION
**Pull request by yanmxa**
Fixed: https://github.com/stolostron/backlog/issues/23922
Fixed: https://github.com/stolostron/backlog/issues/23923

Signed-off-by: myan [myan@redhat.com](mailto:myan@redhat.com)

### CODE REVIEW

File: test/pkg/test/hoh-policy_test.go [hunk 1]
```diff
@@ -115,7 +122,7 @@ var _ = Describe("policy", Ordered, Label("policy"), func() {
 		}, 5*60*time.Second, 5*1*time.Second).ShouldNot(HaveOccurred())
 	})
 
-	It("add the label for anohter managedcluster to vertify the placement", func() {
+	It("add the label for anothter managedcluster to vertify the placement", func() {
```
Comment (morvencao) [inline test/pkg/test/hoh-policy_test.go:125]: s/vertify/verify/

File: test/pkg/test/hoh-application_test.go [hunk 2]
```diff
@@ -83,11 +86,15 @@ var _ = Describe("application", Label("app"), Ordered, func() {
...
+		By("Print result after adding the label")
+		managedClusters := getManagedCluster(httpClient, token)
+		printClusterLabel(managedClusters)
 	})
 
-	It("create the application/subscription", func() {
+	It("deploy the application/subscription", func() {
 		By("Apply the appsub to labeled cluster")
-		_, err := clients.Kubectl(utils.HUB_OF_HUB_CLUSTER_NAME, "apply", "-f", APP_SUB_YAML)
+		_, err := clients.Kubectl(clients.HubClusterName(), "apply", "-f", APP_SUB_YAML)
```
Comment (morvencao) [inline test/pkg/test/hoh-application_test.go:97]: We're using `kubectl` binary to apply the resources?

Comment (yanmxa) [reply test/pkg/test/hoh-application_test.go:97]: Yeah, we use the `kubectl` by the `oc/exec` plugin.

**Pull request merged**